sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -839,6 +839,24 @@ object SQLConf {
     .checkValues(HiveCaseSensitiveInferenceMode.values.map(_.toString))
     .createWithDefault(HiveCaseSensitiveInferenceMode.NEVER_INFER.toString)
 
+  val HIVE_BLOBSTORE_SUPPORTED_SCHEMES =
+    buildConf("spark.sql.hive.blobstore.supported.schemes")
+      .doc("Comma-separated list of supported blobstore schemes.")
+      .version("3.1.0")
+      .stringConf
+      .transform(_.toLowerCase(Locale.ROOT))
+      .createWithDefault("s3,s3a,s3n")
```

Comment:
I a bit hesitate to use vendor-specific values here...  This optimization works well only for s3 systems and how about any other blob stores? If it works well, I think its better to make the change more general and we don't need to strictly follow the Hive configurations. WDYT? @HyukjinKwon @dongjoon-hyun

Reply:
On second thoughts, just setting an empty string here looks okay.

Reply:
`s3`, `s3a`, `s3n` are just default values, which are used in Apache Hive as well. https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L5025

I am okay to make it empty string. My concern is that it will be bit difficult for end users to specify what kind of values here.
If the concern can be acceptable, then I will make it empty string.

Reply:
>  My concern is that it will be bit difficult for end users to specify what kind of values here if we remove default values.

I think how-to-use should be described in  a property `.doc`. The python script automatically generates documents based on these properties: See https://spark.apache.org/docs/3.0.0-preview/configuration.html .

Reply:
Got it. Modified.